src/strings/PowerFxResources.en-US.resx

```diff
@@ -5158,4 +5158,15 @@
     <value>Can't delegate {0}: contains a behavior function '{1}'.</value>
     <comment>Warning message.</comment>
   </data>
+  <data name="AboutPlainText" xml:space="preserve">
+    <value>The PlainText function removes HTML and XML tags.</value>
```

Comment:
>The PlainText function removes HTML and XML tags. [](http://example.com/codeflow?start=11&length=49)

We should keep the existing description (unless it is buggy): "Removes all formatting and returns a plain text value."

Reply:
Updated. thanks